src/auth/auth.controller.ts

```diff
@@ -17,11 +17,12 @@ export class AuthController {
   signOut() {
     return this.authService.signOut();
   }
-  @UseGuards(RefreshJwt)
+
+  @UseGuards(RefreshJwtGuard)
   @Get('refreshToken')
   getRefreshTokens(@Request() req: any) {
```

Comment:
the function name is bad, think about the real feature this function do, is that get or refresh token ?